pyomyo.py

```diff
@@ -43,6 +43,8 @@
 import sys
 import threading
 import time
+import numpy as np
+import pandas as pd
```

Comment:
suck it punk

Reply:
test 1:22

---
**Sema Reaction:** :trophy: This code is awesome